Code/Framework/AzCore/AzCore/Math/Aabb.cpp

```diff
@@ -238,13 +238,23 @@ namespace AZ
 
     void Aabb::ApplyMatrix3x4(const Matrix3x4& matrix3x4)
     {
-        const AZ::Vector3 extents = GetExtents();
-        const AZ::Vector3 center = matrix3x4 * GetCenter();
-        AZ::Vector3 newHalfExtents(
-            0.5f * matrix3x4.GetRowAsVector3(0).GetAbs().Dot(extents),
-            0.5f * matrix3x4.GetRowAsVector3(1).GetAbs().Dot(extents),
-            0.5f * matrix3x4.GetRowAsVector3(2).GetAbs().Dot(extents));
-        m_min = center - newHalfExtents;
-        m_max = center + newHalfExtents;
+        Vector3 a, b, axisCoeffs;
+
+        Vector3 newMin, newMax;
+        newMin = newMax = matrix3x4.GetTranslation();
```

Comment:
uber nit: If you have a passing AR run you can ignore this and this also likely isn't in the coding standards, so this is can totally be ignored, but I've seen it recommended (by clang-tidy checks if I recall) to avoid declaring multiple variables on the same line and to avoid chaining assignments (this is purely to make the code more readable and avoid for want of a better word, 'cuteness').

```c++
Vector3 a;
Vector3 b;
Vector3 axisCoeffs;

Vector3 newMin = matrix3x4.GetTranslation();
Vector3 newMax = newMin;
```

I'd also maybe suggest using different variable names other than `a` and `b` (that is in the coding standards as avoid using single letter variable names - maybe some math code is exempt but this I think could have more useful names). I'd also sugges `j` for the loop index could be `axis` or `axisIndex` instead.

I also think moving the `Vector` declarations inside the loop (closer to first use) might be nice too (so they can be const and get initialized directly instead of being left uninitialized to start). I don't think this should hurt performance in any meaningful way.

(feel free to ignore all this if you think I'm talking rubbish 😅)

Reply:
Addressed all of the above for both ApplyTransform and ApplyMatrix3x4 so that they remain consistent.

Reply:
Absolute legend! 😎 Thank you kind sir! 🫶